### PR TITLE
 Avoid unnecessary child processes (faster training)

### PR DESCRIPTION
**Pull request by stweil**

### CODE REVIEW

**Comment by stweil** [comment]
@wrznr, did you ever try Tesseract training with the GT4HistOCR ground truth (or know someone who did)?

**Comment by wrznr** [comment]
@stweil I am pretty sure @Doreenruirui did!

**Comment by stweil** [comment]
@jbaiter, `dta19/1882-keller_sinngedicht/04970.nrm.png` from GT4HistOCR is broken. `convert`cannot read it, and it looks like [this](https://ub-backup.bib.uni-mannheim.de/~stweil/04970.nrm.png) in the browser.

**Comment by Doreenruirui** [comment]
> @jbaiter, `dta19/1882-keller_sinngedicht/04970.nrm.png` from GT4HistOCR is broken. `convert`cannot read it, and it looks like [this](https://ub-backup.bib.uni-mannheim.de/~stweil/04970.nrm.png) in the browser.

I have not encounter this problem before because I did not train it on the whole GT4HistOCR data.

**Comment by wrznr** [comment]
There is a way to pass seeds to `sort -R`:
```shell
seq 1 100 | sort -R --random-source=<(openssl enc -aes-256-ctr -pass pass:"42" -nosalt </dev/zero 2>/dev/null)
```
where *42* is the seed. This may make the additional script superfluous.

**Comment by wrznr** [comment]
@stweil I thought I saw a commit but now it's gone.

**Comment by stweil** [comment]
I accidentally pushed that commit here, but it is unrelated to this pull request, so I fixed my mistake and removed it again.

It is still there, see #71.

**Comment by wrznr** [comment]
It is still very useful! We are waiting for another PR ;)

**Comment by lokesh-stack** [comment]
can some one help me with this error: 

lstmtraining \
  --traineddata data/foo/foo.traineddata \
  --net_spec "[1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c" \
  --model_output data/checkpoints/foo \
  --learning_rate 20e-4 \
  --train_listfile data/list.train \
  --eval_listfile data/list.eval \
  --max_iterations 10000
Warning: given outputs 0 not equal to unicharset of 64.
Missing ] at end of [Series]!
Failed to create network from spec: [1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c**
Makefile:163: recipe for target 'data/checkpoints/foo_checkpoint' failed

**Comment by wrznr** [comment]
@lokesh-stack Pls. do not file issues as comments in already merged PRs. File an issue at https://github.com/tesseract-ocr/tesstrain/issues instead. Pls. also remember to provide the necessary details for reproducing your error. Otherwise we cannot be of assistance.

**Comment by stweil** [comment]
@lokesh-stack, please update your local installation.
